Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -125,6 +114,8 @@ CSequenceBatchRenderDialog::CSequenceBatchRenderDialog(float fps, QWidget* pPare
     m_renderTimer.setInterval(0);
     m_renderTimer.setSingleShot(true);
 
+    m_skipFramesCount = REGISTER_INT("TrackViewSkipFramesCount", 30, 0, "Set the number of frames to skip during the warm up and switching to the Game Mode");
```

Comment:
If I'm not mistaken the CryEngine CVar system is now superseeded by the AzCore IConsole, so you should use AZ_CVAR or something along these lines (and yeah should be made more explicit and removed in existing code)